### ISSUE TITLE
Notifications is not readable

### ISSUE THREAD

**Issue by aabdelzaher** [SEGUC17/Foobar#43]
1: Severity: ( low )

2:Reported: by Abdelzaher

3:Description: When viewing my alerts the date is written in ISO format which is not very user friendly. Also when the title is long enough it will overlap with the date and will be unreadable.

4: Steps to reproduce the issue: Click on message icon (alerts) in the home page.
5: Expected result: The date should be displayed in a user friendly way, and when the title is long it should be displayed in an appropriate way that it will not overlap with the date.

**Comment by aabdelzaher** [comment]
![alerts](https://cloud.githubusercontent.com/assets/18190340/25586787/81420968-2e90-11e7-87e3-7ef5cc129ae0.png)

**Comment by markarsanious** [comment]
Yeah , you view them from the view announcements in the navbar available to you.
